website/docs/r/ssmcontacts_contact.html.markdown

```diff
@@ -45,14 +45,14 @@ resource "aws_ssmcontacts_contact" "example" {
 
 The following arguments are required:
 
-- `alias` - (Required) A unique and identifiable alias for the contact or escalation plan.
+- `alias` - (Required) A unique and identifiable alias for the contact or escalation plan. 1-50 characters, a-z, 0-9, _ (underscore), and - (hyphen).
 
 - `type` - (Required) The type of contact engaged. A single contact is type PERSONAL and an escalation
   plan is type ESCALATION.
 
 The following arguments are optional:
 
-- `display_name` - (Optional) Full friendly name of the contact or escalation plan.
+- `display_name` - (Optional) Full friendly name of the contact or escalation plan. 1-255 characters, alphanumeric, _ (underscore), - (hyphen), and spaces.
```

Comment:
Similar tweaks to phrasing here. I noted in the [API documentation](https://docs.aws.amazon.com/incident-manager/latest/APIReference/API_SSMContacts_CreateContact.html) that periods are also accepted.

```suggestion
- `display_name` - (Optional) Full friendly name of the contact or escalation plan. If set, must be between 1 and 255 characters, and may contain alphanumerics, underscores (`_`), hyphens (`-`), periods (`.`), and spaces.
```